xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -5767,6 +5767,72 @@ TEST_P(ParameterizedFp8GemmRewriteTest,
       )");
 }
 
+TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABUnscaledDWithAllGatherF8) {
+#if CUDA_VERSION < 12000
+  GTEST_SKIP() << "F8 gemm rewrite is only supported in CUDA 12 and above.";
+#endif
+  HloModuleConfig config = GetModuleConfigForTest();
+  if (config.num_partitions() != 4 || !config.use_spmd_partitioning()) {
+    GTEST_SKIP() << "All-gather is only expected in SPMD.";
+  }
```

Comment:
It looks like `GetModuleConfigForTest` defaults to using 1 partition:

https://github.com/openxla/xla/blob/2e64c449d90b1005a5beb8079e406a7b56fdae5d/xla/tests/hlo_test_base.h#L193-L201

I don't think it enables SPMD by default either. So I don't think this test will ever run

Reply:
Resolved.